app/lib/frontend/templates/views/consent/package_uploader_invite.mustache

```diff
@@ -0,0 +1,17 @@
+{{! Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<p>
+  <code>{{ active_account_email }}</code>
```

Comment:
The variable `active_account_email` sounds like it is the email that is currently active (as in currently signed in).

However, the text says that it is the email that sent you the invitation. This is verify confusing.

Could we change this to `initiating_account_email` or `inviting_account_email` or ??? Even `invite_initiating_account_email` might work, long but descriptive :)